assets/stylesheets/_components.scss

```diff
@@ -403,3 +403,4 @@
 @import 'signup/validation-fieldset/style';
 @import 'support/support-user/style';
 @import 'vip/vip-logs/style';
+@import 'blocks/plan-thank-you-card/style'
```

Comment:
We try to keep this list sorted alphabetically. Could you move it to the right place, please? :)